pkg/api/mlflow/api/request/expiriment.go

```diff
@@ -0,0 +1,57 @@
+package request
+
+// ExperimentTagPartialRequest is a partial request object for different requests.
+type ExperimentTagPartialRequest struct {
+	Key   string `json:"key"`
+	Value string `json:"value"`
+}
+
+// ExperimentPartialRequest is a partial request object for different requests.
+type ExperimentPartialRequest struct {
+	ID               string                        `json:"experiment_id"`
+	Name             string                        `json:"name"`
+	ArtifactLocation string                        `json:"artifact_location"`
+	LifecycleStage   string                        `json:"lifecycle_stage"`
```

Comment:
The type should be `LifecycleStage`

Reply:
there is no reason to have a type here. It is just a string. So you can probably put any string and it will be valid. We have to check that during the validation process to guarantee, that  `LifecycleStage` has correct value.

Reply:
Understood!